media/css/m24/components/footer-refresh.scss

```
@@ -316,21 +295,57 @@ $max-footer-content-width: $content-max;
     }
 
     .mzp-js-language-switcher-select {
-        background: $m24-color-alt-white url('/media/img/icons/m24-small/down-caret-white.svg') 95px 0.75em no-repeat;
-        background-size: 16px 16px;
+        background-color: transparent;
+        background-image: none;
         border-radius: 0;
-        border: $border-width solid transparent;
-        color: $m24-color-black;
+        border: $border-width solid $m24-color-alt-black;
+        color: $m24-color-alt-black;
         font-weight: 600;
         margin-top: 16px;
-        min-width: unset;
         padding-left: 36px;
-        width: 136px;
+        max-width: 100%;
+        width: fit-content;
+        position: relative;
+
+        &::before {
+            @include bidi(((left, 0, auto), (right, auto, 0)));
+            background: $m24-color-alt-white url('/media/img/icons/m24-small/globe-white.svg') center center no-repeat;
+            background-size: 18px 18px;
+            top: 0;
+            margin-top: 0;
+            content: "";
+            display: block;
+            height: 32px;
+            width: 32px;
+            position: absolute;
+        }
+
+        &::after {
+            @include bidi(((background-position, calc(100% - 8px) 0.75em, 8px 0.75em),));
```

This may be easier as:
```suggestion
            @include bidi(((background-position, right 8px center, 8px center),));
```

---

Great suggestion!